### ISSUE TITLE
When 2 (or more) tables got the same column name, only one is being returned.

### ISSUE THREAD

**Issue by leodutra** [w1nk/node-odbc#65]
Example:

``` sql
SELECT  A.X,
        B.X
FROM    TABLE_A A, 
        TABLE_B B
```

Returns only one of the columns' (`X`) data.
**MUST** return both `A.X` and `B.X`.

**Comment by wankdanker** [comment]
Right. Not sure how exactly to handle this when the result is an object. We'd have to look for duplicate column names and then modify the column name in the resulting object accordingly. I'm not a huge fan of that personally.

You can fetch the columns in array mode like this:

``` js
var db = require('odbc')();
db.fetchMode = db.FETCH_ARRAY;
db.openSync(process.env.ODBC_CONNECTION_STRING);
db.querySync('select 1 as a, 2 as a union select 345 as a, 456 as a');
/*
[
  [ 1, 2 ],
  [ 345, 456 ]
]
*/
```

I have encountered what you are experiencing on other platforms and always thought it was my job as the person querying the database to return unique column names. So, in your example, I would have changed the query to:

``` sql
select A.X as aX,
    B.X as bX
from table_a A, 
    table_b B
```

I personally don't have any interest in implementing a fix for this when fetching in object mode. I'm open to accepting patches though.

Dan
